ddtrace/tracer/civisibility_writer.go

```diff
@@ -30,7 +30,7 @@ var _ traceWriter = (*ciVisibilityTraceWriter)(nil)
 // to the Datadog backend. It manages the payload size and flushes the data when necessary.
 type ciVisibilityTraceWriter struct {
```

Comment:
one last nit: is it trace or eventwriter? 😛

Reply:
I'd keep it as is if you don't mind, since this writer implements the traceWriter interface: https://github.com/DataDog/dd-trace-go/blob/6d9882bf120f4b82d9931f48201af9d10ba1110c/ddtrace/tracer/tracer.go#L237